packages/extension-ui/src/components/contexts.tsx

```diff
@@ -5,20 +5,30 @@
 import { AccountJson, AuthorizeRequest, SigningRequest } from '@polkadot/extension/background/types';
 
 import React from 'react';
+import { AvailableThemes } from '.';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const noop = (to?: string): void => {};
```

Comment:
```suggestion
const noop = (): void => {};
```
Seems to lint & build for me like this, and without the rule disable.